certbot/util.py

```diff
@@ -206,24 +207,20 @@ def check_permissions(filepath, mode, uid=0):
     return misc.compare_file_modes(file_stat.st_mode, mode) and file_stat.st_uid == uid
 
 
-def safe_open(path, mode="w", chmod=None, buffering=None):
+def safe_open(path, mode="w", chmod=None):
     """Safely open a file.
 
     :param str path: Path to a file.
     :param str mode: Same os `mode` for `open`.
-    :param int chmod: Same as `mode` for `os.open`, uses Python defaults
+    :param int chmod: Same as `mode` for `security.open`, uses Python defaults
         if ``None``.
-    :param int buffering: Same as `bufsize` for `os.fdopen`, uses Python
-        defaults if ``None``.
 
     """
     open_args = ()  # type: Union[Tuple[()], Tuple[int]]
     if chmod is not None:
         open_args = (chmod,)
     fdopen_args = ()  # type: Union[Tuple[()], Tuple[int]]
-    if buffering is not None:
```

Comment:
Why don't we need this anymore? What was this doing?

Reply:
It was doing nothing, in fact. This option is never used in certbot.